### DISCUSSION TITLE
Nouvelle version du prototype disponible!

### DISCUSSION THREAD

**Discussion by remillc** [bibudem/coquille-web#9] [Announcements]
Une nouvelle version (v3.0.0) du prototype vient d'être mise en ligne. Il s'agit d'une version majeure puisqu'on a délaissé le _framework_ `Astro` pour celui de `Gatsby`.

Vous devrez installer les sources à nouveau, pour plus d'information, consultez le fichier README.md.

[https://github.com/bibudem/prototype-web](https://github.com/bibudem/prototype-web)

**Comment by msevigny** [comment]
Ça fonctionne tel que documenté de mon côté!

**Reply by remillc** [reply]
Il y a une précision à apporter quand à la mise à jour des sources locales: il faut d'abord demander la dernière version des sources en premier! Donc, dans l'ordre:

```
git pull
npm install
npm run dev
```

En fait, c'est toujours une bonne chose que de débuter une session de travail par un `git pull && npm install`. De cette façon, on s'assurer de toujours travailler à partir de la dernière version des fichiers.
